model/host/hostutil.go

```diff
@@ -1019,7 +1019,7 @@ func (h *Host) CheckTaskDataFetched(ctx context.Context, env evergreen.Environme
 						return false, nil
 					}
 					return true, errors.New("fetching task data not finished")
-				}, 5, time.Second, 15*time.Second)
+				}, 10, 5*time.Second, time.Minute)
```

Comment:
The durations seem fine, but it doesn't make a whole lot of sense to me that we have backoff/jitter for this since we're just waiting for an external command to finish. Can you change this to a simple timer with the same timeout?